lib/collection.js

```diff
@@ -141,6 +141,16 @@ Collection.prototype.remove = function (query, opts, cb) {
   })
 }
 
+Collection.prototype.rename = function (name, opts, cb) {
+  if (typeof opts === 'undefined') return this.rename(name, {}, noop)
+  if (typeof cb === 'undefined') return this.rename(name, noop)
```

Comment:
It's more safe to not rely on `typeof` and instead just write `if (!opts) return ...` - that way it will work if people pass in `null` and other falsy values as well.